src/ansys/simai/core/api/sse.py

```diff
@@ -144,6 +144,12 @@ def _handle_data_model_event(self, data):
                     logger.debug(
                         f"Unknown action {target['action']} of type {target['type']} received for job or resource event. Ignoring."
                     )
+        elif target["type"] == "geomai_model":
+            self.simai_client.geomai.models._handle_sse_event()
+        elif target["type"] == "geomai_prediction":
+            self.simai_client.geomai.predictions._handle_sse_event()
+        elif target["type"] == "geomai_training_data":
+            self.simai_client.geomai.training_data._handle_sse_event()
```

Comment:
Shouldn't it be 
```suggestion
        elif target["type"] == "geomai_model":
            self.simai_client.geomai.models._handle_sse_event(data)
        elif target["type"] == "geomai_prediction":
            self.simai_client.geomai.predictions._handle_sse_event(data)
        elif target["type"] == "geomai_training_data":
            self.simai_client.geomai.training_data._handle_sse_event(data)
``` 
?
Got errors : `Error handling SSE event: Directory._handle_sse_event() missing 1 required positional argument: 'data'. `